esrally/telemetry.py

```diff
@@ -1408,6 +1409,199 @@ def record(self):
             self.metrics_store.put_doc(doc, level=MetaInfoScope.cluster, meta_data=data_stream_metadata)
 
 
+class IngestPipelineStats(InternalTelemetryDevice):
```

Comment:
As discussed with @pquentin - it probably makes most sense for this to be enabled by default given there's 0 risk of results skew with the current approach of collecting before and after a benchmark.